README.md

```diff
@@ -0,0 +1,3 @@
+# PrototypeAndela
+Andela Drills
+[![Build Status](https://travis-ci.org/knite51/PrototypeAndela.svg?branch=chore%2Ftravisci-install)](https://travis-ci.org/knite51/PrototypeAndela)
```

Comment:
Let this be the first thing at the top of the file, Before the `PrototypeAndela`

Reply:
Also, go back to where you copied it from, and ensure that the branch you are copying is the develop branch.

Reply:
Okay. But how do i edit my read me file. i couldnt find any button that says 'edit'

Reply:
Found it!

Reply:
@knite51 You edit it locally. On your system. There is the README file in your code.

Reply:
okay but to do this. I would have to make a pull origin request on this branch to my local files. Because i create the README file on github itself. Asking for future cases.